en/frames_plane/wing_wing_z84.md

```diff
@@ -71,8 +71,22 @@ The images below give a rough idea about the assembly process, which is simple a
 ![wing wing build09](../../assets/airframes/fw/wing_wing/wing_wing_build09.jpg)
 ![Wing Wing Z-84 build](../../assets/airframes/fw/wing_wing/wing_wing_build11.jpg)
 
-## Airframe Configuration
+## PX4 Configuration
+
+### Airframe Configuration
 
-Select the Z-84 in the flying wing section of the QGC airframe config: 
+Select **Flying Wing > Generic Flying Wing** in the QGroundControl [Airframe Configuration](../en/config/airframe.md): 
 
 ![QGC - select firmware for West Wing](../../assets/airframes/fw/wing_wing/qgc_firmware_flying_wing_west_wing.png) 
+
+### Actuator Mapping
+
+Set up the [Actuator Configuration](../en/config/actuators.md) to match the wiring for the ailerons and throttle as [indicated above](#wiring).
+
+![QGC - set the actuators](../../assets/airframes/fw/wing_wing/qgc_actuator_config.png)

```

Comment:
@sfuhrer Can you sanity check this image. It should match the config up in the wiring section.

Reply:
Yes looks correct!